packages/turbo-graph-ui/lib/turbo.ts

```diff
@@ -62,14 +62,22 @@ async function executeCommand(
 
 async function findTurboConfig(currentPath = '.'): Promise<Data> {
   const files = await fs.readdir(currentPath)
-  const turboConfig = files.find((file) => file === 'turbo.json')
+  const turboConfig = files.find((file) => file === 'turbo.json' || file === 'turbo.jsonc')
   const continueSearch = () => findTurboConfig(`..${path.sep}${currentPath}`)
   if (!turboConfig) {
     return continueSearch()
   }
   const file = `${currentPath}${path.sep}${turboConfig}`
-  const buffer = await fs.readFile(file)
-  const config = JSON.parse(buffer.toString())
+  const contents = await fs.readFile(file, { encoding: 'utf8' })
+  const config = await (async () => {
+    try {
+      return JSON.parse(contents)
+    } catch {
+      const jsoncParser = await import('jsonc-parser')
```

Comment:
not sure whether bundle size really matters for a local app/whether you care about it, but figured a dynamic import here would at least make it possible to code split this

Reply:
I don't think bundle size is of importance here and changed the implementation to always parse with the jsonc-parser.